packages/credentials/src/status-list-credential.ts

```diff
@@ -1,47 +1,57 @@
 import pako from 'pako';
 import { getCurrentXmlSchema112Timestamp } from './utils.js';
 import { VerifiableCredential, DEFAULT_VC_CONTEXT, DEFAULT_VC_TYPE, VcDataModel } from './verifiable-credential.js';
-import type { ICredentialStatus} from '@sphereon/ssi-types';
 import { Convert } from '@web5/common';
 
 export const DEFAULT_STATUS_LIST_VC_CONTEXT = 'https://w3id.org/vc/status-list/2021/v1';
 export const DEFAULT_STATUS_LIST_VC_TYPE = 'StatusList2021Credential';
 
+/**
+ * The status purpose dictated by Status List 2021 spec.
+ * @see {@link https://www.w3.org/community/reports/credentials/CG-FINAL-vc-status-list-2021-20230102/#statuslist2021entry | Status List 2021 Entry}
+ */
 export enum StatusPurpose {
-  REVOCATION = 'revocation',
-  SUSPENSION = 'suspension',
+  revocation = 'revocation',
+  suspension = 'suspension',
 }
 
 /**
  * The size of the bitstring in bits.
  * The bitstring is 16KB in size.
  */
-const BITSTRING_SIZE = 16 * 1024 * 8; // 16KB in bits
+const BITSTRING_SIZE = 16 * 1024 * 8; // 16KiB in bits
 
 /**
  * StatusListCredentialCreateOptions for creating a status list credential.
  *
  * @param statusListCredentialId The id used for the resolvable path to the status list credential [String].
  * @param issuer The issuer URI of the credential, as a [String].
  * @param statusPurpose The status purpose of the status list cred, eg: revocation, as a [StatusPurpose].
- * @param issuedCredentials The credentials to be included in the status list credential, eg: revoked credentials, list of type [VerifiableCredential].
+ * @param revokedCredentials The credentials to be included in the status list credential, eg: revoked credentials, list of type [VerifiableCredential].
  */
 export type StatusListCredentialCreateOptions = {
   statusListCredentialId: string,
   issuer: string,
   statusPurpose: StatusPurpose,
-  issuedCredentials: VerifiableCredential[]
+  revokedCredentials: VerifiableCredential[]
```

Comment:
A credential can be revoked, suspeneded, or any other noun actually. Status Purpose should actually probably just be a string

Reply:
Good point, though the action of suspend or revoke result in the same bit flip to 1, this would probably go away if you decide to take my other PR suggestion of have `revoke()`, for now I'll rename to `credentialToDisable`. Happy to discuss but I don't think `includedCredentials` is it.